src/helpers/embed.ts

```diff
@@ -9,78 +9,87 @@ import type { IOpenAiImage } from "btbot-types";
 import type { Message } from "discord.js";
 export class Embed {
 	static success(description: string, title?: string) {
-		const embed = new MessageEmbed();
+		const embed = new EmbedBuilder();
 		embed.setColor(Colors.green);
 		title && embed.setTitle(title);
 		embed.setDescription(description);
 		return embed;
 	}
 
 	static error(description: string, title?: string) {
-		const embed = new MessageEmbed();
+		const embed = new EmbedBuilder();
 		embed.setColor(Colors.red).setTitle(title ?? "Error");
 		embed.setDescription(description);
 		return embed;
 	}
 }
 
 export const sendPaginatedCommandList = async (commands: ICommand[], msg: Message) => {
-	const embeds = constructCommandsEmbedArray(commands);
-	await new Embeds()
-		.setArray(embeds)
-		.setAuthorizedUsers(msg.author.id)
-		.setChannel(msg.channel as TextChannel)
-		.setTimeout(60000)
-		.build();
-};
+	console.log(
+		commands.map((c) => {
+			return { name: c.command, value: c.description };
+		})
+	);
+	const pagEmbed = new PaginatedEmbed({
+		itemsPerPage: 10,
+		paginationType: "field",
+		showFirstLastBtns: false,
+		origUser: msg.author.id
+	})
+		.setTitles(["Commands"])
+		.setFields(
+			commands.map((c) => {
+				return { name: c.command, value: c.description };
+			})
+		);
+
+	await pagEmbed.send({ options: { channel: msg.channel as TextChannel } });
 
-const constructCommandsEmbedArray = (commands: ICommand[]) => {
-	const commandsPerPage = 10;
-	const pageCount = Math.ceil(commands.length / commandsPerPage);
-	return commands.reduce((acc, { command, description }) => {
-		if (!command) return acc;
-		const lastEmbed = acc[acc.length - 1];
-		const commandsAdded =
-			Math.max(acc.length - 1, 0) * commandsPerPage + lastEmbed?.fields.length || 0;
-		if (commandsAdded % commandsPerPage === 0) {
-			const embed = new MessageEmbed();
-			const pageNumber = Math.ceil(commandsAdded / commandsPerPage) + 1;
-			embed
-				.setColor(Colors.green)
-				.setTitle(`Commands (${pageNumber}/${pageCount})`)
-				.addField(command, description);
-			acc.push(embed);
-			return acc;
-		}
-		lastEmbed.addField(command, description);
-		return acc;
-	}, [] as MessageEmbed[]);
+	// const embeds = constructCommandsEmbedArray(commands);
```

Comment:
No need to leave this commented-out stuff around now that we have the paginated embeds working for `!help` and `!album`, right?